src/plots.jl

```diff
@@ -528,6 +526,8 @@ mutable struct MatrixPlot <: Plot
             end
             writedlm(filename,out)
         end
+        xrange = xrange .+ (-step(x)/2, step(x)/2)
```

Comment:
I don't think `x` is defined here. See Travis failure.

Reply:
This code is not relevant anymore but I thought I had already tested the case `raster = true` by passing it as argument to the constructor but after inspecting https://github.com/JuliaTeX/PGFPlots.jl/blob/163bbe49d9227192eadba8385d3977f8e8424e1f/src/plots.jl#L499-L504 I see that this was not the case. Perhaps `raster = true` should not be ignored for `length(A) <= 1e4`.